packages/editor/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+## 3.0.0 (Unreleased)
+
+- Breaking: The `wideAlign` block supports hook has been removed. Use `alignWide` instead.
```

Comment:
Shouldn’t it also be included in blocks? Sometimes those changes touch more than one package.

Reply:
Hmm, I agree it's relevant for someone implementing a block (interfacing with `blocks`), but the logic itself lives in `editor`, and as part of a general workflow I'd be conscious of removing the "make me think" part involved with considering documenting cascading effects.

Which is all to say: I don't think we should include it in `blocks`.